cfme/fixtures/artifactor_plugin.py

```diff
@@ -150,12 +150,12 @@ def fire_art_hook(config, hook, **hook_args):
     if client is None:
         assert UNDER_TEST, 'missing artifactor is only valid for inprocess tests'
     else:
-        client.fire_hook(hook, **hook_args)
+        return client.fire_hook(hook, **hook_args)
```

Comment:
**question** where are those used - i believe i'm missing a link in chain there

Reply:
This is now in a different PR(https://github.com/ManageIQ/integration_tests/pull/7763). Where I commented(https://github.com/ManageIQ/integration_tests/pull/7763#issuecomment-418812041) why this needs to return value.